sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLWindowFunctionSuite.scala

```diff
@@ -367,4 +367,50 @@ class SQLWindowFunctionSuite extends QueryTest with SQLTestUtils with TestHiveSi
         | select * from v2 order by key limit 1
       """.stripMargin), Row(0, 3))
   }
+
+  test("lead/lag should return the default value if the offset row does not exist") {
```

Comment:
oh, originally we used window functions from Hive. It should be reason that we put this file at here. Let me move it.